src/sardana/tango/core/util.py

```diff
@@ -1156,9 +1156,16 @@ def prepare_logging(options, args, tango_args, start_time=None,
                 os.makedirs(path, 0o777)
 
             from sardana import sardanacustomsettings
-            maxBytes = getattr(sardanacustomsettings, 'LOG_FILES_SIZE', 1E7)
-            backupCount = getattr(sardanacustomsettings, 'LOG_BCK_COUNT', 5)
 
+            tangods = args[0]
+            
+            if tangods == "Pool":
```

Comment:
if arvg[0] isn't one of "Pool" or "MacroServer", this could lead to some confusing errors. Can it ever be something else? What du you get if you launch with "Sardana"?

Reply:
good arg, I will follow up with @vanesil and @CarlaCTaka

Reply:
`args[0]` can only be: `Pool`, `MacroServer` or `Sardana`

We pushed the changes, can you take a look ?